pkg/actions/action.go

```diff
@@ -106,20 +108,11 @@ func (a *Action) Validate() error {
 }
 
 func (a *Action) Match(spec MatchSpec) (bool, error) {
-	// at least one matched event definition
-	var actionOn *ActionOn
-	switch spec.EventType {
-	case graveler.EventTypePreCommit:
-		actionOn = a.On.PreCommit
-	case graveler.EventTypePreMerge:
-		actionOn = a.On.PreMerge
-	case graveler.EventTypePostCommit:
-		actionOn = a.On.PostCommit
-	case graveler.EventTypePostMerge:
-		actionOn = a.On.PostMerge
-	default:
+	if !supportedEvents[spec.EventType] {
```

Comment:
This check is actually unnecessary, since the spec received from graveler - it will always be a valid graveler.EventType

Reply:
Thanks! Done.